### ISSUE TITLE
[Extension]: civitai helper buttons not showing up since a committ

### ISSUE THREAD

**Issue by gsgoldma** [vladmandic/sdnext#1557]
### Issue Description
Missing buttons that civitai extension enables, like prompt of image, keywords, and a link back to the civitai website model link.

see post here, https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper/issues/206

### URL link of the extension

https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper

### URL link of the issue reported in the extension repository
https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper/issues/206

_No response_

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by brknsoul** [comment]
You might want to add https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper/issues/206 to the "URL link of the issue reported in the extension repository" section of your report to make this a valid report.

**Comment by gsgoldma** [comment]
> You might want to add [butaixianran/Stable-Diffusion-Webui-Civitai-Helper#206](https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper/issues/206) to the "URL link of the issue reported in the extension repository" section of your report to make this a valid report.

thanks i did

**Comment by gsgoldma** [comment]
Yeah @gsgoldma, I checked @goldmojo and no updates there. I haven't had time to look into these breaking changes (I'm on Vlad's auto1111 implementation) for my fork. The number of recent updates has been tough to keep up with. The new Diffusers back-end option, for example, damn near broke everything for me.

One solution was recommended to change the line in a styles.css file.

However, this fork does not have that file. Is there some equivalent thing to change in this repo?

I use this workaround in webui, not in the extension :
=> Edit stable-diffusion-webui/style.css
=> Add this lines on top (line 13 in my style.css file)

.extra-network-thumbs .actions .additional {
    position: relative;
    }

It may resolve the issue.

**Comment by vladmandic** [comment]
don't ever edit `styles.css`, add that into `user.css` if you really want.

**Comment by gsgoldma** [comment]
https://github.com/HeartlessAUS/Stable-Diffusion-Webui-Civitai-Helper this fork works for now if anyone is wondering

**Comment by AndrewWHales** [comment]
Hi, these look to be broken again? I am using a fresh git pull of the main SD-Next branch and a fresh install via extensions tab of https://github.com/zixaphir/Stable-Diffusion-Webui-Civitai-Helper and the only icon on the network lora previews is "Get Details".

**Comment by vladmandic** [comment]
closing as will-not-fix, neither variants of civitai-helper have been updated this year and differences to make them work with sdnext are too big.
